<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMain</class>
 <widget class="QMainWindow" name="QMain">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>541</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Agricultural Management Software --- LVP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="ControlBar">
      <item>
       <widget class="QPushButton" name="buttonExit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSettings">
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="mainScreen" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="layoutFieldManagement">
         <item>
          <widget class="QLabel" name="titleFields">
           <property name="text">
            <string>&lt;b&gt;Field Management&lt;/b&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listFields"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="layoutButtonsFieldManagement">
           <item>
            <widget class="QPushButton" name="buttonNewField">
             <property name="text">
              <string>New Field</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonEditField">
             <property name="text">
              <string>Edit Field</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonDeleteField">
             <property name="text">
              <string>Delete Field</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutFieldData" stretch="0,0,0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;b&gt;Field Data&lt;/b&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbGeneralData">
           <property name="title">
            <string>General Data</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Name: </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbl_Name">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Size(ha): </string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Registered Number: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="lbl_Size">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="lbl_RegNumber">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbDataCrops">
           <property name="title">
            <string>Crops</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="1" column="1">
             <widget class="QLabel" name="lbl_Crop1">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="lbl_InterCrop2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="lbl_Crop2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="lbl_InterCrop3">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="lbl_Crop3">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="lbl_InterCrop4">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="lbl_Crop4">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLabel" name="lbl_InterCrop5">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QLabel" name="lbl_Crop5">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lbl_InterCrop1">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_HarvestYear1">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lbl_HarvestYear2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="lbl_HarvestYear3">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="lbl_HarvestYear4">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="lbl_HarvestYear5">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>24</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSettings</sender>
   <signal>clicked()</signal>
   <receiver>QMain</receiver>
   <slot>openSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>QMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>QMain</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNewField</sender>
   <signal>clicked()</signal>
   <receiver>QMain</receiver>
   <slot>newField()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDeleteField</sender>
   <signal>clicked()</signal>
   <receiver>QMain</receiver>
   <slot>deleteField()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonEditField</sender>
   <signal>clicked()</signal>
   <receiver>QMain</receiver>
   <slot>editField()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openSettings()</slot>
  <slot>wantExit()</slot>
  <slot>save()</slot>
  <slot>newField()</slot>
  <slot>deleteField()</slot>
  <slot>editField()</slot>
 </slots>
</ui>
